tests/clients/test_alerts.py

```diff
@@ -283,22 +283,3 @@ def test_get_all_posts_expected_data(self, mock_session, user_context):
             and posted_data["srtKey"] == "key"
             and posted_data["srtDirection"] == "ASC"
         )
-
-    def test_get_by_name_filters_correct_record(self, mock_get_all_session, user_context):
```

Comment:
These tests arent needed anymore because they test filtering that happens _after_ we get the response, Which will no longer be the case (the filtering will be the responsibility of the alert api now)